### DISCUSSION TITLE
Jedi support

### DISCUSSION THREAD

**Discussion by fithisux** [posit-dev/positron#3918] [Feedback]
In VSCodium I'm able to configure Jedi as a language server for Python. This is not currently possible for Python in Positron.

**Comment by juliasilge** [comment]
Positron uses the Jedi LSP for Python as part of the built-in Python extension. Can you share more about the kind of configuration you are not able to set up?

**Reply by fithisux** [reply]
Nice to listen to this. Normally the Python extension in VScodium has a drop down to change between Pylance and Jedi. I did not find it. So I assumed that something else is used. Thank you for sharing this information and now I do not have any hesitation to use Positron for my Python Projects.
